Assignment1/src/test/java/AssignmentMain.java

```diff
@@ -0,0 +1,107 @@
+import io.restassured.RestAssured;
+import io.restassured.path.json.JsonPath;
+import static io.restassured.RestAssured.*;
+import static org.hamcrest.MatcherAssert.assertThat; 
+import static org.hamcrest.Matchers.*;
+import io.restassured.path.json.JsonPath;
+
+public class AssignmentMain {
+	
+	static String baseURI;
+	public static void main(String args[]) {
+		
+		// get users request getCall 1
+
+		baseURI = RestAssured.baseURI = "https://reqres.in/";
+		String list_users = given().queryParam("page", 2)
+		.when()
+		.get("/api/users?page=2")
+		.then().assertThat().statusCode(200)
+		.extract().response().asString();
+		
+		System.out.println("List users getCall 1");
+		System.out.println(list_users);
+		
+		// get single user getCall 2
+		int id;
+		id = 2;
+		
+		
+		String singleUser = given().queryParam("id", id)
```

Comment:
You can move each such test in its own method and then call that method from the main method. That will make the code more readable as you can give meaningful names to the methods. e.g. this specific test could be in a method called `test_get_single_user_returns_http_200`.


A further improvement could be to have one class for each api and then in that class you have one method for each test case. In that setup, this test and the next one would go in the same class but two different methods.

Reply:
moved tests in methods and created and grouped test in different classes